semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -85,7 +86,8 @@ class Main(settings: Settings, reporter: Reporter) {
                 }
               }
               result.foreach { infos =>
-                index.append(infos)
+                val uri = infos.classfile.relative + ".semanticdb"
```

Comment:
This is duplicated with the code above, why remove the method? Seems like a regression rather than improvement.